docs/source/gcp/contribute.rst

```diff
@@ -148,98 +148,29 @@ Each resource has to have test cases. There are implemented test cases for resou
 Testing
 =========
 
-Test cases for resources list methods
---------------------------------------
+c7n_gcp follows the same guideance for test authoring as the :ref:`core c7n code
+base<Creating Tests>`. Examples with relative directories (for example, ``tests/foo/bar``)
+should be applied to ``tools/c7n_gcp`` and not the root of the cloud-custodian
+repository.
 
-To create a test case for `list` method is used following scenario.
+In order to execute live tests you will need to set two additional environment
+variables. These can be either set in your IDE or exported on the command line.
+The first is ``GOOGLE_CLOUD_PROJECT`` - this should point to a valid Google
+Cloud Project you have access to. The second, ``GOOGLE_APPLICATION_CREDENTIALS``
+should be the corresponding credentials json with access to the aforementioned
+project.
 
-- A factory is created based on recording real data from a GCP project resource.
+For example:
 
-    .. code-block:: python
-
-        factory = self.record_flight_data(<name of a file>, project_id=project_id)
-
-The `name of a file` means the folder name that has JSON file(s) with expected response(s) on the request from a testing policy.
```

Comment:
why did all this content get removed?

Reply:
Seemed duplicated by the Developer Contributor guide for writing tests

Reply:
fair, it might be useful to link back to that content.

Reply:
oh.. i see the rst ref back there already, cool